src/Table/Column/Link.php

```diff
@@ -136,7 +136,11 @@ public function getDataCellTemplate(\atk4\data\Field $field = null)
 
         $label = '';
         if ($this->use_label) {
-            $label = $field ? ('{$' . $field->short_name . '}') : '[Link]';
+            if ($this->use_label === true) {
+                $label = $field ? ('{$' . $field->short_name . '}') : '[Link]';
```

Comment:
`{$` is there for some templating?

Reply:
yes, $label is a template string

Reply:
then it is set wrong - label can be be used as template...

Reply:
I didn't get your comment?

Reply:
imagine label with `{$xx}`

Reply:
This is true for any decorator, then you need to escape the template like syntax

Reply:
and html? `<script>alert("hi, my Bitcoin wallet is...");</script>`

Reply:
scripts gets directly executed

Reply:
I really hate these single purpose solution and even more if they are not complete/secure at all.

![image](https://user-images.githubusercontent.com/2228672/89630008-05fba900-d89f-11ea-92e5-e9ba632127dd.png)

can we solve it better? is this for some core atk4 feature?

Reply:
The additional code is due to BC notices, etc. Actually additional code is 1 line.
It provides much more flexible functionality for the Link where dev can set whatever template he likes, not just the field value

Reply:
but implemented as single purpose if - should we copy paste that 100x times?!?